magma/loong.patch

```diff
@@ -0,0 +1,89 @@
+diff --git a/PKGBUILD b/PKGBUILD
+index 2ef55de..b9fc950 100644
+--- a/PKGBUILD
++++ b/PKGBUILD
+@@ -7,7 +7,7 @@
+ 
+ _pkgname=magma
+ pkgbase=$_pkgname
+-pkgname=(magma-cuda magma-hip)
```

Comment:
You'd better append `pkgname=(${pkgname[@]/magma-cuda})` to the tail of PKGBUILD instead of simply delete `magma-cuda` from here. The line you change is next to `pkgver`, which offen changes, and it will make the patch rotten frequently. See https://wszqkzqk.github.io/2024/08/22/loongarchlinux-port-tips/#%E5%AF%B9pkgbuild%E5%A4%A7%E6%AE%B5%E5%88%A0%E9%99%A4%E7%9A%84%E5%A4%84%E7%90%86.